mathics_scanner/data/named-characters.yml

```diff
@@ -10428,6 +10446,7 @@ Venus:
   wl-unicode-name: FEMALE SIGN
 
 VerticalBar:
+  ascii: '|'
```

Comment:
WMA uses `\mid` for the LaTeX representation. However, here there is a Pytest that does not allows to have two symbols with the same `amslatex` entry.

Reply:
In short, it is okay to adjust the pytest. But `\mid` doesn't look like the right character used in https://reference.wolfram.com/language/ref/VerticalBar.html  . To me it looks more like `\shortmid`. See  http://perso.neel.cnrs.fr/olivier.cepas/aassymbols.pdf

The longer discussion... 

As it says in https://tex.stackexchange.com/questions/52901/why-is-mid-so-called , this symbol is a long vertical bar with "thick space" on either side.  Currently, it is used for `\[Divides]` and this seems right. 

As for adjusting the test,  There does not need to be a one-to-one correspondence between a named character and a symbol. We have these "RawXXX' characters which generally are the same as some other character.

Reply:
> In short, it is okay to adjust the pytest. But `\mid` doesn't look like the right character used in https://reference.wolfram.com/language/ref/VerticalBar.html . To me it looks more like `\shortmid`. See http://perso.neel.cnrs.fr/olivier.cepas/aassymbols.pdf
> 

OK, but this is a third-order correction. Now I am trying to have complete enough tables to make that the documentation compiles. Then we can choose the optimal latex command.